community-events/content/challenge-project/03-exercise-portal-deploy.md

```diff
@@ -0,0 +1,162 @@
+# ☁️ Exercise 3: Deploy on Azure Static Web App
+
+In this exercise, you will learn how to deploy the `portal` project on Azure using the Azure Static Web Apps CLI and GitHub Actions. You'll need to use your Azure account for this exercise. If you don't have an account, create a free [Azure Free Trial](https://azure.microsoft.com/free/?WT.mc_id=academic-101248-cyzanon) or use the [GitHub Student Developer Pack — GitHub Education](https://aka.ms/Copilot4Students) program.
+
+## Preparing the scenario
+
+### Create a new branch
+
+First, you will need to create a new branch in your GitHub repository and follow some configuration steps. To do this, run the following commands in the Visual Studio Code terminal in Codespaces:
+
+```bash
+  git checkout -b portal-deploy
+```
+
+### Create a copy and rename
+
+Create a copy of the `portal` folder and rename it to `portal-deploy` This duplication is necessary so that we can deploy the portal scenario separately from other parts of the project, such as the `blog` and `api`.
+
+```bash
+  cp -r packages/portal packages/portal-deploy
+```
+
+With the `portal-deploy` folder created, access the file `packages/portal-deploy/package.json` and change the `name` property to `portal-deploy`.
+
+![Package.json](./images/new-portal-package-json.jpg)
+
+## Navigate and delete existing files
+
+Now, go back to the terminal, as we will run some routines with basic Linux commands and configure the project deployment with the Azure Static Web Apps CLI.
+
+```bash
+  cd packages/portal-deploy
+```
+
+We need to delete the `packages/portal/node_modules` folder and the `swa-cli.config.json` file. To do this, we will use the linux command `rm -rf`. The -r is used to delete folders and the -f is used to force deletion.
+
+**Remove the node_modules folder:**
```

Comment:
Not sure why removing/reinstalling dependencies is needed? SWA CLI is already part of the project dependencies too